experimental/framework/frs-client/tsconfig.json

```diff
@@ -12,6 +12,8 @@
         "src/**/*"
     ],
     "exclude": [
+        "dist",
```

Comment:
these don't need to be added here since they're part of `ts-common-config.json` above already (which I think is what's breaking policy check for you)

Reply:
I realized this after I pushed the commit. Was hoping to remove it before anyone got to it. You're too quick to review 😂